crates/noirc_evaluator/src/ssa_refactor/ir/printer.rs

```diff
@@ -148,6 +150,9 @@ pub(crate) fn display_instruction(
         Instruction::Call { func, arguments } => {
             writeln!(f, "call {}({})", show(*func), value_list(function, arguments))
         }
+        Instruction::ForeignCall { func, arguments } => {
+            writeln!(f, "call {}({})", func, value_list(function, arguments))
```

Comment:
Do we want to print this maybe as foreign_call?

Reply:
Changed it